scripts/bindings.patch

```diff
@@ -1,12 +1,12 @@
-diff --git a/crates/utils/src/core/allocationmanager.rs b/crates/utils/src/core/allocationmanager.rs
+diff --git a/crates/utils/src/slashing/core/allocationmanager.rs b/crates/utils/src/slashing/core/allocationmanager.rs
```

Comment:
You can regenerate this by:

1. emptying this file
2. generating bindings
3. fixing any compile issues
4. running `git diff crates/utils > scripts/bindings.patch`

I should probably write this down somewhere...